tls/s2n_psk.c

```diff
@@ -163,3 +217,87 @@ int s2n_psk_verify_binder(struct s2n_connection *conn, struct s2n_psk *psk,
 
     return S2N_SUCCESS;
 }
+
+static S2N_RESULT s2n_psk_write_binder(struct s2n_connection *conn, struct s2n_psk *psk,
+        const struct s2n_blob *binder_hash, struct s2n_stuffer *out)
+{
+    ENSURE_REF(binder_hash);
+
+    struct s2n_blob binder;
+    uint8_t binder_data[S2N_TLS13_SECRET_MAX_LEN] = { 0 };
+    GUARD_AS_RESULT(s2n_blob_init(&binder, binder_data, binder_hash->size));
+
+    GUARD_AS_RESULT(s2n_psk_calculate_binder(psk, binder_hash, &binder));
+    GUARD_AS_RESULT(s2n_stuffer_write_uint8(out, binder.size));
+    GUARD_AS_RESULT(s2n_stuffer_write(out, &binder));
+
+    return S2N_RESULT_OK;
+}
+
+static S2N_RESULT s2n_psk_write_binder_list(struct s2n_connection *conn, const struct s2n_blob *partial_client_hello,
+        struct s2n_stuffer *out)
+{
+    ENSURE_REF(conn);
+    ENSURE_REF(partial_client_hello);
+    ENSURE_REF(out);
```

Comment:
You don't need this here, `s2n_stuffer_reserve_uint16` will already prevent that.